families.json

```
@@ -126,7 +126,7 @@
   "BIZ UDPMincho": [],
   "Babylonica": ["scrawl", "calligraphic", "elegant"],
   "Bad Script": ["script"],
-  "Bagel Fat One": [],
+  "Bagel Fat One": ["korean", "fat", "cute", "rounded"],
```

Small change as the library has "round" tag:

```suggestion
  "Bagel Fat One": ["korean", "fat", "cute", "round"],
```